x-pack/plugins/apm/public/components/app/service_dashboards/actions/save_dashboard_modal.tsx

```diff
@@ -73,7 +73,10 @@ export function SaveDashboardModal({
 
   const {
     path: { serviceName },
-  } = useApmParams('/services/{serviceName}/dashboards');
+  } = useAnyOfApmParams(
+    '/services/{serviceName}/dashboards',
+    '/mobile-services/{serviceName}/dashboards'
+  );
```

Comment:
I kind of don't like this `useAnyOfApmParams` hook. It is the start of a lot of if-elses that may make this component confused. Soon enough we'll have "if services" or "if mobile-services".

Can't we instead receive the serviceName from the parents and remove the use of the route hook?

Reply:
updated here: https://github.com/elastic/kibana/pull/174255/commits/477b21ef6a3d5c147e28d7b233beefcc387517c5